contrib/llvm-project-cmake/CMakeLists.txt

```diff
@@ -1,12 +1,15 @@
 if (APPLE OR SANITIZE STREQUAL "undefined" OR SANITIZE STREQUAL "memory")
-   set (ENABLE_EMBEDDED_COMPILER_DEFAULT OFF)
+    # llvm-tblgen, that is used during LLVM build, doesn't work with UBSan.
+    set (ENABLE_EMBEDDED_COMPILER_DEFAULT OFF)
+    set (ENABLE_DWARF_PARSER_DEFAULT OFF)
 else()
-   set (ENABLE_EMBEDDED_COMPILER_DEFAULT ON)
+    set (ENABLE_EMBEDDED_COMPILER_DEFAULT ${ENABLE_LIBRARIES})
```

Comment:
Made ENABLE_LIBRARIES=0 imply ENABLE_EMBEDDED_COMPILER=0 by default. Hope there isn't a good reason why it wasn't done this way originally.